### PR TITLE
[OH GOD OH FUCK] [TESTMERGE ONLY] SECURITY BALLISTICS - Just gonna pull and hijack Trilby's PR for now

### PR DESCRIPTION
**Pull request by silicons**
Adds .22 to security, most of the code's from @Trilbyspaceclone 

Opening this so I can edit natively.

### CODE REVIEW

**Comment by Useroth** [comment]
Iiiiii am not sure how I feel about security having ballistics round-start available to every sec officer, considering sec gear is OP as fuck already, on top of scaling with gear from R&D.

They have lethals in the armory. Sec officers don't need to be able to handle every threat imaginable with just their standard everyday carry gear, especially considering the balance of quite a couple of things like the d-esword relies in good part on security having mainly energy-projectile weapons.

**Comment by Trilbyspaceclone** [comment]
@kevinz000  Cat Man forgets to link OG PR
#8527

File: code/game/objects/items/AI_modules.dm [hunk 1]
```diff
@@ -369,7 +369,7 @@ AI MODULES
 	if(!targName)
 		return
 	subject = targName
-	laws = list("You may not injure a [subject] or, through inaction, allow a [subject] to come to harm.",\
+	laws = list("You may not injure a [subject], cause a [subject] to come to harm, or, through inaction, allow a [subject] to suffer critical condition inducing harm, loss of limb or life, or permanent functional incapacitation.",\
```
Comment (Ghommie) [inline code/game/objects/items/AI_modules.dm:372]: Unstated, out-of-scope, unrelated change. Make a different PR for it, Thanks.

File: code/datums/ai_laws.dm [hunk 2]
```diff
@@ -29,7 +29,7 @@
 /datum/ai_laws/default/asimov
 	name = "Three Laws of Robotics"
 	id = "asimov"
-	inherent = list("You may not injure a human being or, through inaction, allow a human being to come to harm.",\
+	inherent = list("You may not injure a human, cause a human to come to harm, or, through inaction, allow a human to suffer critical condition inducing harm, loss of limb or life, or permanent functional incapacitation.",\
```
Comment (Ghommie) [inline code/datums/ai_laws.dm:32]: Ditto.

**Review by Ghommie — COMMENTED** [review]
The `22x19p` icon_state is not aligned with the `22x19p-7` and `22x19p-0` ones, ergo a few pixels off set to the left.
